Makefile

```diff
@@ -55,6 +55,12 @@ ifeq ($(OS),Linux)
 	install -d -m 0750 $(DESTDIR)/etc/auto.master.d
 	install -m 0640 autofs/auto.master.d/gandi.autofs $(DESTDIR)/etc/auto.master.d/
 	install -m 0750 autofs/auto.gandi $(DESTDIR)/etc/
+
+	install -d ${DESTDIR}/etc/kernel/postinst.d
+	install -m 0750 ./etc/kernel/postinst.d/gandi-decompress-kernel ${DESTDIR}/etc/kernel/postinst.d/
+	install -d ${DESTDIR}/usr/local/sbin
+	install -m 0750 ./usr/local/sbin/decompress-kernel ${DESTDIR}/usr/local/sbin/
```

Comment:
Why /usr/local/sbin and not /usr/sbin ?

Reply:
I wasn't sure which one to use, but it's fixed now.